birdhouse/components/cowbird/config/cowbird/config.yml.template

```diff
@@ -0,0 +1,155 @@
+## Combined configuration definition to setup objects at Cowbird startup
+## See documentation 'Configuration' for more details on each section.
+
+## See example: https://github.com/Ouranosinc/cowbird/blob/master/config/config.example.yml
+## See details: https://pavics-cowbird.readthedocs.io/en/latest/configuration.html#file-config-yml
+
+
+# [Required] Definition of managed services by Cowbird.
+# FIXME: rename 'services' section to 'handlers' and refactor (issue https://github.com/Ouranosinc/cowbird/issues/22)
+services:
+  Magpie:
+    active: true
+    url: https://${PAVICS_FQDN_PUBLIC}/magpie
+    admin_user: ${MAGPIE_ADMIN_USERNAME}
+    admin_password: ${MAGPIE_ADMIN_PASSWORD}
+  Geoserver:
+    active: true
+    url: http://${PAVICS_FQDN_PUBLIC}:8765/geoserver
+    workspace_dir: ${WORKSPACE_DIR}
+    admin_user: ${GEOSERVER_ADMIN_USER}
```

Comment:
Side note, as this PR is for integration and not 100% functionality. Happened on a brand-new IaC instance too.

`${GEOSERVER_ADMIN_USER}` somehow is not being templated correctly.

The other variables, except `WORKSPACE_DIR` which has not been defined yet, are replaced as expected in `config.yml`. 

I had to add `GEOSERVER_ADMIN_USER` to Cowbird's own `.../components/cowbird/default.env` for it to be replaced properly, which is weird considering there's no problem with `GEOSERVER_ADMIN_PASSWORD` and `PAVICS_FQDN_PUBLIC`

Might be a side effect from something else in the configuration? I'm bringing this up in case it is reproduced in subsequent PRs where configurations will be fine-tuned.

Reply:
Good catch @francisPLT `GEOSERVER_ADMIN_PASSWORD` is templated expanded properly because it is in the `VARS` list here https://github.com/bird-house/birdhouse-deploy/blob/d1ecc63284ec9d2940bfa2b1b4baca3fbe1308b3/birdhouse/pavics-compose.sh#L31

`GEOSERVER_ADMIN_USER` is not in that `VARS` list so that's why it has not been template expanded properly.  The reason it is not in that list is because it currently is not needed be template expanded anywhere.

The fix is simple, just add it to the same location as `GEOSERVER_ADMIN_PASSWORD`.

Reply:
Variable added.